design/Proposal For View Definition.md

```diff
@@ -0,0 +1,84 @@
+﻿
+
+
+
+## Proposal to extend the View Definition
+**Background**
+The current  interface for a SqlView  which is a first class object in Nessie specifies a Sql text and a Dialect  and schema :
+
+    public interface SqlView extends Contents {  
+      
+      public static enum Dialect {  
```

Comment:
- What criteria to use to accept a dialect to be added? I guess the goal is interoperability but if lots of dialects are supported, it's getting harder and harder to reach that goal, with only some dialects to be supported by some tools in practice? For example how a view using a Dremio/Presto dialect be used from a spark context?
- Should dialect be versioned too? Like Hive 2.x and Hive 3.x dialects (assuming they are different)?